math_hurdlerActivity.py

```diff
@@ -88,8 +84,14 @@ def _stop_play_cb(self, button):
             button.set_tooltip(_("Stop"))
 
     def read_file(self, file_path):
-        self.game.read_file(file_path)
-
+        with open(file_path, 'r') as f:
+            restore_data = f.read()
+            restore_data = str(restore_data).split()
+            score, play_state = [restore_data[i] for i in (0, 1)]
+            self.game.restore_game(score, play_state)
+            
     def write_file(self, file_path):
-        # self.game.write_file(file_path)
-        pass
+        score, play_state = self.game.write_file()
+        with open(file_path, 'w') as f:    
+            f.write(f'{score}\n{play_state}')
+  
```

Comment:
Could you remove the no new line at end of file?

Reply:
done